src/internet_identity/tests/integration/archive_integration.rs

```diff
@@ -349,13 +340,27 @@ mod pull_entries_tests {
             caller: principal_1(),
             sequence_number: 4,
         };
-        assert_eq!(
+        assert_eq_with_lenient_timestamp(
             entries.entries.get(4).unwrap().as_ref().unwrap(),
-            &delete_entry
+            &delete_entry,
         );
         Ok(())
     }
 
+    fn assert_eq_with_lenient_timestamp(actual: &Entry, expected: &Entry) {
```

Comment:
nit: maybe add a comment explaining why that helper and why the logic accepting 1 sec. difference?